src/components/shoppingListItem/shoppingListItem.test.tsx

```diff
@@ -0,0 +1,75 @@
+import { ReactElement } from 'react'
+import { describe, test, expect } from 'vitest'
+import { renderAuthenticated } from '../../support/testUtils'
+import {
+  gamesContextValue,
+  shoppingListsContextValue,
+} from '../../support/data/contextValues'
+import { GREEN } from '../../utils/colorSchemes'
+import { PageProvider } from '../../contexts/pageContext'
+import { GamesContext } from '../../contexts/gamesContext'
+import { ShoppingListsContext } from '../../contexts/shoppingListsContext'
+import { ColorProvider } from '../../contexts/colorContext'
+import ShoppingListItem from './shoppingListItem'
+
+const renderInContexts = (ui: ReactElement) =>
```

Comment:
Technically speaking, the `ShoppingListItem` doesn't consume any of these contexts. However, within the next few PRs it will, and when that happens it will be nice to not have to go back through the test file adding them to all the tests.